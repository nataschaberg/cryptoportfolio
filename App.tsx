/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * Generated with the TypeScript template
 * https://github.com/emin93/react-native-template-typescript
 *
 * @format
 */

import React, {Fragment} from 'react';
import {
  StyleSheet,
  View,
  Text,
} from 'react-native';

const App = () => {
  return (
    <Fragment>
      <View style={styles.parentContainer}>
        <View style={styles.innerContainer}>
          <Text>Initialized</Text>
        </View>
      </View>
    </Fragment>
  );
};

const styles = StyleSheet.create({
  parentContainer: {
    height: '100%',
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: 'white'
  },
  innerContainer: {
    backgroundColor: 'yellow' 
  }
});

export default App;