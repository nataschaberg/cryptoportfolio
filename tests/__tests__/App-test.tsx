/**
 * @format
 */

import 'react-native'
import React from 'react'
import App from '../../App'

// Note: test renderer must be required after react-native.
import * as ShallowRenderer from 'react-test-renderer/shallow'

describe('App Component', () => {
  it('Should render correclty', () => {
    const shallow = ShallowRenderer.createRenderer()
    const app = shallow.render(<App />)
    expect(app).toMatchSnapshot()
  })
})
