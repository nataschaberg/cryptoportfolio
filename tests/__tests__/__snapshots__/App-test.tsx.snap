// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Should render correclty 1`] = `
<React.Fragment>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "height": "100%",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "yellow",
          "marginVertical": "2%",
          "width": "100%",
        }
      }
    >
      <Overview />
    </View>
  </View>
</React.Fragment>
`;
